app/static/js/searchUser.js

```diff
@@ -27,7 +28,11 @@ export default function searchUser(inputId, callback, clear=false, parentElement
                 // value: "{column: response, column2: response2, ...}"
                 // must JSON.parse
                 label: (item[1]["firstName"]+" "+item[1]["lastName"]+" ("+item[0]+")"),
-                value: JSON.stringify(columnDict)
+                dictvalue:JSON.stringify(columnDict),
```

Comment:
Is there a reason this was changed?

Reply:
Yes, that has solved the issue of the whole JSON dictionary showing in the search bar.